code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -495,3 +497,79 @@
 
 /obj/item/crusher_trophy/wolf_ear/on_mark_detonation(mob/living/target, mob/living/user)
 	user.apply_status_effect(/datum/status_effect/speed_boost, 1 SECONDS)
+
+//cosmetic items for changing the crusher's look
+
+/obj/item/crusher_trophy/retool_kit
+	name = "crusher sword retool kit"
+	desc = "A toolkit for changing the crusher's appearance without affecting the device's function. This one will make it look like a sword."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "retool_kit"
+	denied_type = /obj/item/crusher_trophy/retool_kit
+	var/retool_icon = "crusher_sword"
+	var/retool_inhand_icon = "crusher_sword"
+	var/retool_projectile_icon = "pulse1"
```

Comment:
dont forget to autodoc these

Reply:
What would you recommend I add for documentation here?

Reply:
```suggestion
	///new icon state for our crusher's world icon
	var/retool_icon = "crusher_sword"
	///new icon state for how our crusher appears in hands
	var/retool_inhand_icon = "crusher_sword"
	///icon state for projectiles fired by our crusher 
	var/retool_projectile_icon = "pulse1"
```
maybe something along the lines of this, or anything that u feel would adequately explain this to future devs that stumble upon this file